docs/source/installation.rst

```diff
@@ -0,0 +1,146 @@
+package dependencies. These virtual environments contain distinct python interpreters, allowing packages
+installed within a given environment to be independent of the system interpreter.
+To install conda, go to the following website, and install the version
+needed for your operating system (OS).
+
+https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html#installing-conda-on-a-system-that-has-other-python-installations-or-packages
+
+General Commands
+~~~~~~~~~~~~~~~~
+To create conda environments, run the following
+
+``conda env create -n <name of environment> python=<desired python version>``
+
+For example, if I wanted to create an environment named ``mesmerize``, with python version 3.9,
+you can run:
+
+``conda env create -n mesmerize python=3.9``
+
+To see a list of existing conda environments, run the following:
+
+``conda env list``
+
+To activate a specific environment, run the following:
+
+``source activate <name of environment>``
+
+Git Commands
+----------------
+Git commands allow you to access, clone, modify, and update github repositories. To install
+Git on your computer, go to the following website, and install the version needed for your
+operating system (OS).
+
+https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+**Note for Windows Users**
+
+Windows powershell gives me many issues when trying to install git within the powershell terminal,
+so I would recommend installing Git Bash Terminal instead, and running git commands from with Git Bash.
+
+To download, go to the following website and follow the directions
+
+https://www.educative.io/answers/how-to-install-git-bash-in-windows
+
+Package Installation Manager (PIP)
+------------------------------------
+PIP is a package manager for python, meaning it's a tool
+that allows you to install and manage libraries and dependencies.
+You can use pip to install packages within a specific virtual environment
+(i.e. conda environments), thus creating an isolated python interpreter
+with only the libraries you need for your project.
+To install pip, go to the following weibsite, and install the version needed for your operating system (OS).
+
+https://pip.pypa.io/en/stable/installation/
+
+General Commands
+~~~~~~~~~~~~~~
+To see a list of existing packages and their versions, run the following:
+
+``pip list``
+
+To install a the latest stable version of a specific package, run the following:
+
+``pip install <package name>``
+
+For example, if I need to install numpy, I can run the following:
+
+``pip install numpy``
+
+To install a specific version of a package run the following:
+
+``pip install <package name> == <version #>``
+
+For example, if I need to install version 1.22 of numpy, I can run the following:
+
+``pip install numpy==1.22``
+
+To install packages within a specific conda environment, just active the conda environment and then run
+your pip commands
+
+Installation
```

Comment:
put all instructions in a single code block so they can be copy-pasted, for example: https://github.com/nel-lab/mesmerize-core#for-development

Reply:
no they're in separated headings, just have all the install commands in a single code block